src/drivers/char/terminal/vtparse.c

```diff
@@ -46,10 +46,11 @@ static void do_action(struct vtparse *parser, vt_action_t action, char ch) {
 		break;
 
 	case VT_ACTION_COLLECT:
-		if (token->attrs_len == -1) {
+		// Buffer overflow check may be needed
+		//if (token->attrs_len == -1) {
```

Comment:
@AntonKozlov I'm terribly confused by this. I thout that attr_len is unsigned but it seems that it have type `char`. But you can see how clang raises a warning above (I just added this).